spec/dummy/app/models/authentication.rb

```diff
@@ -0,0 +1,7 @@
+class Authentication < ApplicationRecord
+  include PasswordConcern
+  include TenancyConcern
+  encrypt_column :password
```

Comment:
Minor, but it reads better to keep this line with the mixin inclusion since they are related.